ts-eks-helm/klotho.yaml

```diff
@@ -0,0 +1,9 @@
+app: ts-eks-helm
+provider: aws
+path: .
+out_dir: compiled
+execution_units:
+    main:
+        type: eks
+        helm_chart_options:
+            install: true
```

Comment:
not blocking, but i dont think we need this one since we have the one in the test/ dir